src/noisepy/seis/utils.py

```diff
@@ -24,3 +27,49 @@ def fs_join(path1: str, path2: str) -> str:
         return posixpath.join(path1, path2)
     else:
         return os.path.join(path1, path2)
+
+
+class TimeLogger:
```

Comment:
Just thinking out loud - Can we add this class as part of `rse_tooklit` package in the future?

Reply:
Yes :)